lldb/include/lldb/Symbol/ObjectFile.h

```diff
@@ -81,9 +81,12 @@ class ObjectFile : public std::enable_shared_from_this<ObjectFile>,
   enum BinaryType {
     eBinaryTypeInvalid = 0,
     eBinaryTypeUnknown,
-    eBinaryTypeKernel,    /// kernel binary
-    eBinaryTypeUser,      /// user process binary
-    eBinaryTypeStandalone /// standalone binary / firmware
+    eBinaryTypeKernel,            /// kernel binary
+    eBinaryTypeUser,              /// user process binary,
+                                  /// dyld addr
+    eBinaryTypeUserAllImageInfos, /// user process binary,
+                                  /// dyld_all_image_infos addr
+    eBinaryTypeStandalone         /// standalone binary / firmware
```

Comment:
```suggestion
    /// kernel binary
    eBinaryTypeKernel, 
    /// user process binary, dyld addr
    eBinaryTypeUser,        
    /// user process binary, dyld_all_image_infos addr
    eBinaryTypeUserAllImageInfos, 
    /// standalone binary / firmware
    eBinaryTypeStandalone         
```

Doxygen requires `///<` for inline comments like this. I don't know how that works with multiline commetns and given our 80 col limit, documenting them above the value is more readable anyway.